llvm/include/llvm/IR/RepoTicket.h

```diff
@@ -51,9 +55,10 @@ void set(GlobalObject *GO, DigestType const &D);
 /// GO does not contain the ticket metadata.
 std::pair<DigestType, bool> get(const GlobalObject *GO);
 
-struct GONumber {
-  unsigned FuncNum = 0;
-  unsigned VarNum = 0;
+enum class Tags : char {
```

Comment:
This could also be inside the new class as it is only used in there (AFAICT).

Reply:
Moved inside the new class.